src/test/java/com/mattbertolini/liquibase/logging/slf4j/Slf4jLoggerTest.java

```diff
@@ -20,6 +20,7 @@
 import liquibase.logging.LogMessageFilter;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.stubbing.OngoingStubbing;
```

Comment:
Is this import needed. I don't see any other changes.

Reply:
good catch, will remove it.